### PR TITLE
RLP-1077: add readme

### PR DESCRIPTION
**Pull request by mortelli**
This PR solves issue [RLP-1077](https://jirainfuy.atlassian.net/browse/RLP-1077), which is adding a `README.md` file to this project.

It also renames the `sample-topologies` folder to `topologies`.

Please preview the file through [this link](https://github.com/mauricioirace/lumino-tests/tree/feature/RLP-1077%2Fadd-readme).

### CODE REVIEW

File: test/cases/channels.test.ts [hunk 1]
```diff
@@ -22,90 +21,122 @@ describe("Channels", () => {
...
-      amountOnWei: toWei(0),
-      rskPartnerAddress: nodes.target.client.address
-    });
-    await nodes.target.client.sdk.closeChannel({
-      tokenAddress: getTokenAddress(Token.LUM.toString()),
-      partnerAddress: nodes.initiator.client.address
-    });
-  }, 60 * 1000);  
-
+  it(
```
Comment (mortelli) [inline test/cases/channels.test.ts:24]: looks like some automatic formatting changes took place here

File: test/cases/payments.test.ts [hunk 2]
```diff
@@ -21,66 +21,72 @@ describe("Payments", () => {
...
-      partnerAddress: nodes.target.client.address
-    });
-    const targetChannel = await nodes.target.client.sdk.getChannel({
-      tokenAddress: getTokenAddress(Token.LUM.toString()),
-      partnerAddress: nodes.initiator.client.address
-    });
-    expect(initiatorChannel.balance === OldInitiatorChannel.balance);
-    expect(targetChannel.balance === OldTargetChannel.balance);
-  }, TIMEOUT);
+  it(
```
Comment (mortelli) [inline test/cases/payments.test.ts:24]: here as well

File: test/cases/bootstrapping.test.ts [hunk 3]
```diff
@@ -1,51 +1,57 @@
...
-        expect(isEmpty(mauriAddress)).toBe(false);
-        expect(isEmpty(jonaAddress)).toBe(false);
-    }, TIMEOUT);
-
-    afterAll(async () => {
-       await basicEnvironment.stop();
-    }, TIMEOUT);
-
-})
+  let basicEnvironment: LuminoTestEnvironment;
```
Comment (mortelli) [inline test/cases/bootstrapping.test.ts:12]: looks like some automatic formatting changes took place here

File: test/cases/channels.test.ts (hunk 1 above)
Comment (mortelli) [inline test/cases/channels.test.ts:24]: here as well

File: test/cases/payments.test.ts (hunk 2 above)
Comment (mortelli) [inline test/cases/payments.test.ts:24]: here as well

File: README.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+## About
+
+The purpose of the `lumino-tests` project is to provide tools which can run arbitrarily complex automated tests. These aim to serve as complements to the RIF Lumino unit tests, for robustness purposes.
+
+### How it Works
+
+The project is written chiefly in [TypeScript](https://www.typescriptlang.org/) and uses [Jest](https://jestjs.io/) as its testing framework.
+
+Under the hood it makes use of both [Docker](https://www.docker.com/), as well as the [Lumino SDK](https://github.com/rsksmart/lumino-sdk), in order to boot up testing scenarios and carry out actions in them.
```
Comment (mauricioirace) [inline README.md]: We could mention something on [test containers](https://github.com/testcontainers/testcontainers-node)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+- [Node.js](https://nodejs.org/en/)
+- [npm | get npm](https://www.npmjs.com/get-npm)
+- [nvm-sh/nvm: Node Version Manager](https://github.com/nvm-sh/nvm)
+
+#### TypeScript
+
+Install TypeScript by running the following terminal command:
+
+```
+npm install typescript
```
Comment (mauricioirace) [inline README.md]: I think this is not needed

File: test/cases/bootstrapping.test.ts (hunk 3 above)
Comment (mauricioirace) [reply test/cases/bootstrapping.test.ts:12]: mmm husky?

**Review by mauricioirace — APPROVED** [review]
LGTM! Just some minor non-blocking comments

File: README.md (hunk 5 above)
Comment (JONAF2103) [reply README.md]: it shouldn't no, it should be on the package.json file

File: README.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
+<h1 align="center">lumino-tests</h1>
```
Comment (JONAF2103) [inline README.md:1]: Nice adding this readme

File: test/cases/bootstrapping.test.ts (hunk 3 above)
Comment (JONAF2103) [reply test/cases/bootstrapping.test.ts:12]: yup when you push it changes your code, anyway looks good to me, except i don't like the quotes way, i always use a simple quote way for strings

**Review by JONAF2103 — APPROVED** [review]
LGTM just i leave you some comments

File: README.md (hunk 4 above)
Comment (mortelli) [reply README.md]: sorry, i honestly wasn't even aware this was being used :sweat_smile: 

we're using quite a few libraries, so i wanted to mention the bigger ones on the stack.

@JONAF2103 do you think this one can go at the same level as typescript, jest, docker and the lumino SDK?

File: README.md (hunk 5 above)
Comment (mortelli) [reply README.md]: you're right. i remember doing this, but it was actually for the typescript course, not this project.

my bad! fixed

File: test/cases/bootstrapping.test.ts (hunk 3 above)
Comment (mortelli) [reply test/cases/bootstrapping.test.ts:12]: ftr this was done by the widely used [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension.

i suggest we leave it as-is and then settle on js/ts code practices at some point.

unless of course someone has an objection (i'm willing to revert these, it's really no problem)

File: README.md (hunk 4 above)
Comment (JONAF2103) [reply README.md]: Yes, it make sense to me

File: test/cases/bootstrapping.test.ts (hunk 3 above)
Comment (JONAF2103) [reply test/cases/bootstrapping.test.ts:12]: Yes no problem, we should change the rules later if we don't want to use quotes or things like that.

File: README.md (hunk 4 above)
Comment (mortelli) [reply README.md]: added! ty guys for your input

**Pull request merged**